webapplication/notificator/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from publisher.models import Result
+
+# Create your models here.
+
+class Notification(models.Model):
+
+    result = models.OneToOneField(Result, on_delete=models.deletion.CASCADE, primary_key=True)
```

Comment:
Please notify about request being completed, not each result
Please add FK to user
Please add createdat = models.DateTimeField(default=timezone.now)